extensions/wikia/EditPreview/js/editpage.event.helper.js

```diff
@@ -90,10 +90,37 @@ define('editpage.event.helper', ['wikia.window'], function(window, ace){
 		return $('#categories');
 	}
 
+	function iFrameForm(url) {
+		var object = this;
```

Comment:
Consider passing `this` as a additional `context` attribute to the `iFrameForm` function in order to be sure what `this` actually refers to.